lib/screens/create_region_event_screens/components/date_time_widget_row.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+
+class DateTimeWidgetRow extends StatelessWidget {
```

Comment:
Do we need the Widget word in a Widget?
DateTimeWidgetRow --> DateTimeRow

Reply:
fixed 👍